code/modules/mob/living/carbon/xenomorph/abilities/defender/defender_powers.dm

```diff
@@ -13,17 +14,17 @@
 	if(!action_cooldown_check())
 		return
 
-	xeno.crest_defense = !xeno.crest_defense
-
-	if(xeno.crest_defense)
+	if(defender_delegate.crest_lowered == FALSE)
```

Comment:
Don't need to explicitly check for FALSE

Reply:
It's for clarity